src/routes/common/address-info/address-info.helper.ts

```diff
@@ -100,7 +100,10 @@ export class AddressInfoHelper {
       case 'CONTRACT':
         return this.contractsRepository
           .getContract({ chainId, contractAddress: address })
-          .then((c) => new AddressInfo(c.address, c.displayName, c.logoUri));
+          .then(
+            (c) =>
+              new AddressInfo(c.address, c.displayName, c.logoUri ?? undefined),
```

Comment:
In accordance with the potential `null`, this needs to fallback to `undefined` as `AddessInfo` only accepts `string | undefined` as a `logoUri`.

Reply:
Wdyt about leaving a TODO to change the `AddressInfo` constructor to accept `string | null` as a `logoUri` in a different PR?

Reply:
Good point. I've added a comment in ea4c64b57678a9f128a3457f3f6a2ac9ae57e4a6.